examples/basic/versions.tf

```
@@ -2,9 +2,8 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "4.45.0"
+      version = ">= 4.45.0"
```

Can this be unified with the other modules?
```suggestion
      version = ">= 4.30.0"
```